### ISSUE TITLE
Icon request: icon-deaf/deafness/hard-of-hearing

### ISSUE THREAD

**Issue by legobot** [FortAwesome/Font-Awesome#5409]
I believe this is the international symbol for deafness and I think this one would be brilliant to add to the growing list of icons!

![international_symbol_for_deafness](https://cloud.githubusercontent.com/assets/566201/5776175/c225148c-9d7b-11e4-8fd5-65f865ca33f7.jpg)

**Comment by tagliala** [comment]
refers to #4225 and #3726

**Comment by oriadam** [comment]
+ icon-blind

**Comment by oriadam** [comment]
+1

**Comment by crognon** [comment]
+1

**Comment by legobot** [comment]
@oriadam I wonder if font-awesome could add this and create a new group "Accessibility Icons". This will give a much needed boost for the disabled/deaf community online.

**Comment by amanda-phet** [comment]
@legobot I agree that there should be a set of "accessibility icons". They could include icons for screen-reader, keyboard navigation, text-to-speech, and sonification.

**Comment by legobot** [comment]
@amanda-phet We need to bang the drums louder for this to show up in the next release of Font Awesome! 1+

**Comment by arouinfar** [comment]
:+1:  I second @amanda-phet.

**Comment by caduvieira** [comment]
+1

**Comment by caduvieira** [comment]
refers to #4068

**Comment by Levure** [comment]
:+1:
As a deaf programmer, can I say anything else than +1 ? :-)

**Comment by tzusman** [comment]
I just launched [CustomFontIcons.com](https://customfonticons.com/), a place to get crowd-commissioned icons.  All commissioned icons are then released as part of a free font icon set.  A deafness symbol has already been designed and is available in that set.

**Comment by truls1502** [comment]
+1

**Comment by KennetPL** [comment]
+1

**Comment by TerraceMedia** [comment]
+1 

And +1 also for the idea of an accessibility icon set, though I know we're not supposed to ask for whole sets. But especially since FontAwesome is now being used in the [link to US Government Web Design Standards](https://playbook.cio.gov/designstandards/), I think the need will grow.

The [link to Graphics Artists Guild disability access symbols](http://www.graphicartistsguild.org/tools_resources/downloadable-disability-access-symbols0 are one set of sources.

**Comment by codyn329** [comment]
+1

**Comment by MVcodeartbr** [comment]
+1

**Comment by marven89** [comment]
+1

**Comment by codynguyen1116** [comment]
+1

**Comment by AndreLGava** [comment]
+1

**Comment by datakat72** [comment]
+1

We would love an icon that represents deaf and hard of hearing. We support a program like this and need a menu item specific for them. The icon would help easily identify their program.

**Comment by legobot** [comment]
Seeing that the wheelchair icon - http://fortawesome.github.io/Font-Awesome/icon/wheelchair/ - is available. Surely there is a push for "accessibility icons" set soon for 2016?

**Comment by nikkiroylance** [comment]
+1

**Comment by davegandy** [comment]
Added fa-assistive-listening-systems to 4.6.0-wip branch.

**Comment by truls1502** [comment]
Hi @davegandy,

I want to let you know that I am deaf, so it is why the icons around our culture is very important for myself and the deaf community. 

The icon as you mentioned like "fa-assistive-listening-systems", if you are using based of the icons from the website https://www.graphicartistsguild.org/tools_resources/downloadable-disability-access-symbols. 

Actually this icon from the website have a completely different meaning than the icon as @legobot mentioned. Could you add a similar icon with icon-deaf/deafness/hard-of-hearing as @legobot have suggested on this thread?

**Comment by Levure** [comment]
I am deaf too. **I strongly disagree with using "Assistive Listening Systems"** icon to reprensent the deaf community.
Please follow @truls1502 advice !

**Comment by tagliala** [comment]
> Actually this icon from the website have a completely different meaning than the icon as @legobot mentioned. 

Hi @truls1502 , could you please clarify the meaning of both icons?

**Comment by j-mes** [comment]
The icon that @legobot has suggested at the start of this issue thread is the universal icon that will describe deaf as a disability in a clear and concise way. Using anything else different will cause confusion to regarding what the icon's role is and what it's supposed to convey the message to.

If you're looking to use it for assistive hearing devices, use [Graphics Artists Guild](http://www.graphicartistsguild.org/tools_resources/downloadable-disability-access-symbols) for reference to the many different symbols used for clear explanation of icons used to describe disabilities and disability access.

**Comment by truls1502** [comment]
Hi @tagliala, I think that @j-mes explain perfect. 

By the way, could you check this urgent issue as well https://github.com/FortAwesome/Font-Awesome/issues/8895

**Comment by tagliala** [comment]
@j-mes @truls1502 thanks, got it. Reopened this one because they are different icons

**Comment by davegandy** [comment]
@truls1502 @Levure @j-mes Thanks for the clarification. Seriously helpful. I'll add this symbol as well. Apologies for my ignorance. Would you all mind providing more feedback when I get these icons ready? I'm trying to keep them visually cohesive with the rest of Font Awesome but really want to make sure I don't break the meaning.

Seriously. Thanks for the help. Really appreciate it.

**Comment by davegandy** [comment]
@truls1502 @Levure @j-mes @legobot What's the best name for this icon? fa-deaf? fa-deafness? fa-hard-of-hearing? I can also have a couple of aliases.

**Comment by legobot** [comment]
@davegandy Hi! I think **fa-deaf** would be good contender and for aliases **fa-deafness** & **fa-hard-of-hearing**. Thank you so much! It really made my day.

**Comment by truls1502** [comment]
I am agree with @legobot :)

**Comment by davegandy** [comment]
Acceptable?
<img width="800" alt="screenshot 2016-04-07 12 04 46" src="https://cloud.githubusercontent.com/assets/1076721/14359950/83709542-fcb9-11e5-985f-f8d151467554.png">

This pairs with fa-assistive-listening-systems:
<img width="794" alt="screenshot 2016-04-07 12 09 28" src="https://cloud.githubusercontent.com/assets/1076721/14359968/97f12928-fcb9-11e5-9682-301375176252.png">

Thoughts?

**Comment by j-mes** [comment]
@davegandy Your current implementation of the new icons are pretty clear about what they represent and having two noticeable differences which can help communicate with the users when they see either deaf/hard of hearing or hearing assistive devices. :+1:

**Comment by AndreLGava** [comment]
Great, I love this @davegandy , thanks so much for doing this

**Comment by legobot** [comment]
@davegandy Clean and informative. Love it! :+1:

**Comment by davegandy** [comment]
Thanks for the feedback on this, folks! As this is checked into 4.6.0-wip, I'll consider this one closed for the time being. If something doesn't look right, please let me know!
